test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -190,7 +201,7 @@ private static string GetBuildArgs(params string[] args)
             return buildArgs;
         }
 
-        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant)
+        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant, bool isArm)
```

Comment:
Might make sense to default isArm to false.  It would really cleanup the call in CreateTestAppWithSdkImage as it would eliminate the need for named parameters

Reply:
Yes, let's remove named parameters in cases where the number of parameter is low (<3 ?) .  `IsArm` is a bool, which means default is false, right?

Reply:
Updated signature as -

```C#
public static string GetDotNetImage(
            string imageVersion, DotNetImageType imageType, string osVariant, bool isArm = false)
```

Invocation in `CreateTestAppWithSdkImage` as -
```C#
string sdkImage = GetDotNetImage(
                imageDescriptor.DotNetCoreVersion, DotNetImageType.SDK, imageDescriptor.SdkOsVariant);
```